opendbc/car/ford/values.py

```diff
@@ -124,7 +124,14 @@ class CAR(Platforms):
   FORD_ESCAPE_MK4 = FordPlatformConfig(
     [
       FordCarDocs("Ford Escape 2020-22", hybrid=True, plug_in_hybrid=True),
-      FordCarDocs("Ford Kuga 2020-22", "Adaptive Cruise Control with Lane Centering", hybrid=True, plug_in_hybrid=True),
+      FordCarDocs("Ford Kuga 2020-23", "Adaptive Cruise Control with Lane Centering", hybrid=True, plug_in_hybrid=True),
+    ],
+    CarSpecs(mass=1750, wheelbase=2.71, steerRatio=16.7),
+  )
+  FORD_ESCAPE_MK4_5 = FordCANFDPlatformConfig(
+    [
+      FordCarDocs("Ford Escape 2023-24", hybrid=True, plug_in_hybrid=True),
+      FordCarDocs("Ford Kuga 2024", "Adaptive Cruise Control with Lane Centering Assist", hybrid=True, plug_in_hybrid=True),
```

Comment:
The new Kuga in the UK says the following on higher trims:
> Adaptive Cruise Control (with Stop & Go with self-charging and plug-in hybrid powertrains)
> Lane-Keeping Aid with Lane-Departure Warning. Lane Centering Assist included on all self-charging and plug-in hybrid powertrain
They make a distinction between hybrid/non-hybrid, presumably the non-hybrids are only available with manual gearboxes and ACC with Stop and Go and Lane Centering is no available for these

Reply:
TODO: check whether all HEV/PHEV have LCA, or if you can get them with lower trim

Reply:
I think all HEV/PHEV have the required packages

Reply:
By default make sure to use the US advertising/packages, but Kuga is not in the US?

Reply:
the Kuga isn't in the US, and I'm just using US sources for the Escape

Reply:
oh, none of this applies to the Escape, you can still get hybrids without ACC